cmd/client-fs.go

```diff
@@ -1015,25 +1018,34 @@ func (f *fsClient) MakeBucket(region string, ignoreExisting, withLock bool) *pro
 
 // Set object lock configuration of bucket.
 func (f *fsClient) SetObjectLockConfig(mode *minio.RetentionMode, validity *uint, unit *minio.ValidityUnit) *probe.Error {
-	return probe.NewError(APINotImplemented{API: "SetObjectLockConfig", APIType: "filesystem"})
+	return probe.NewError(APINotImplemented{
+		API:     "SetObjectLockConfig",
+		APIType: "filesystem",
+	})
 }
 
 // Get object lock configuration of bucket.
 func (f *fsClient) GetObjectLockConfig() (mode *minio.RetentionMode, validity *uint, unit *minio.ValidityUnit, perr *probe.Error) {
-	return nil, nil, nil, probe.NewError(APINotImplemented{API: "GetObjectLockConfig", APIType: "filesystem"})
+	return nil, nil, nil, probe.NewError(APINotImplemented{
+		API:     "GetObjectLockConfig",
+		APIType: "filesystem",
+	})
 }
 
 // GetAccessRules - unsupported API
 func (f *fsClient) GetAccessRules() (map[string]string, *probe.Error) {
 	return map[string]string{}, probe.NewError(APINotImplemented{
-		API:     "ListBucketPolicies",
+		API:     "GetBucketPolicy",
```

Comment:
Was the API name wrong here?

Reply:
There is no API called ListBucketPolicies?  since its just GetBucketPolicy?